### DISCUSSION TITLE
Pcaclient dont found

### DISCUSSION THREAD

**Discussion by vitelemiux** [winsiderss/systeminformer#992] [Questions / Answers] [answered]
**Hello, PH2 stopped working for me yesterday. I open it, I'm looking for explorer.exe, I double-click on it. Then I click the memory and string
Then I enter 4 Private on, Image on, mapped on I'll wait for the strings to be found. I'll give Filter, Contains (Case-insensitive) I'll look for PcaClient but it won't show me anything there .. Does anyone have any type of solution? Thank you for answer..**

**Comment by dmex** [comment, marked as answer]
https://github.com/processhacker/processhacker/issues/996
